### PR TITLE
eslint errors fix

### PR DESCRIPTION
**Pull request by YashKamboj**
**Description**

This PR fixes #
https://github.com/meshery/meshery/pull/5947
**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/ConnectionWizard/ConfigComponents/Kubernetes.js [hunk 1]
```diff
@@ -29,14 +29,12 @@ const KubernetesConfig = ({
 
   const handleChange = (name) => {
     return (event) => {
+      let fileInput = document.querySelector("#k8sfile");
+      let k8sfile = fileInput.files[0]
```
Comment (Abhishek-kumar09) [inline ui/components/ConnectionWizard/ConfigComponents/Kubernetes.js:33]: did you verified that it is working?

File: ui/components/Navigator.js [hunk 2]
```diff
@@ -590,7 +590,7 @@ class Navigator extends React.Component {
       return (
         <List disablePadding>
           {children.map(({
-            id, onClickCallback, icon, href, title, children
+            id, onClickCallback, icon, href, title, children, show : showc
```
Comment (Abhishek-kumar09) [inline ui/components/Navigator.js:593]: How was then navigator working before? I am just curious.

Comment (YashKamboj) [reply ui/components/Navigator.js:593]: There was no condition when `show` was undefined (i guess), i checked the other places `showc` was used. I also wanted to ask you about it

File: ui/components/ConnectionWizard/ConfigComponents/Kubernetes.js (hunk 1 above)
Comment (YashKamboj) [reply ui/components/ConnectionWizard/ConfigComponents/Kubernetes.js:33]: @Abhishek-kumar09  The successful snackbar comes but it doesn't show up in the list (even without these changes) , would you test it.

**Comment by Abhishek-kumar09** [comment]
I heard that two lint issues are remaing. Would you complete it?

**Comment by Abhishek-kumar09** [comment]
I will test out the changes once completed.

**Review by Abhishek-kumar09 — APPROVED** [review]
Thanks @YashKamboj, LGTM.

**Pull request merged**